modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/sidebar/SidebarContext.es.js

```diff
@@ -25,13 +25,15 @@ const SidebarContextProvider = ({
 		field: null,
 		isOpen: false,
 		totalEntries: 0,
+		type: null,
```

Comment:
@marcosapmf esse atributo `type` é uma string ne? Tem problema inicializar com `''` se for?

Reply:
@carolmariaabb, `type` é uma string sim. Não teria problema da gente inicializá-lo com `''`. Eu só coloquei `null`, porque me baseei no atributo `field`, que também é um string :)

Reply:
Ah, entendi agora. Massa. Melhor deixar assim então 👍